spec/support/factory_girl.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+  config.before :suite do
+    DatabaseCleaner.clean_with :truncation
+  end
+
+  config.before :each do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
+  end
+
+  config.before :each do
+    DatabaseCleaner.start
+    FactoryGirl.reload
```

Comment:
Yea, I put it in there based off of a previous project, we may or may not need it.